### DISCUSSION TITLE
working example of self-host with UI ?

### DISCUSSION THREAD

**Discussion by sandys** [Web3Auth/Web3Auth#409] [Help]
hi,
we were looking to integrate the self-host SDK (https://web3auth.io/docs/sdk/self-host/initialization#example-5) with a firebase login. we are stuck on trying to understand where to integrate the ui.

do u have a working example of using the self-host SDK with a login UI ? 

is this supposed to launch a UI ? we are not able to get it to work. we are on a react-native mobile app.
```

const loginResponse = await (tKey.serviceProvider as TorusServiceProvider).triggerLogin({
    typeOfLogin: "google",
    verifier: "web3auth-testnet-verifier",
    clientId: "134678854652-vnm7amoq0p23kkpkfviveul9rb26rmgn.apps.googleusercontent.com",
});
```

**Comment by himanshuchawla009** [comment]
currently we have this example: https://github.com/tkey/tkey-example

**Reply by tima-t** [reply]
This demo is for web. Is there a similar demo for self-host + react native?

**Reply by ratribiana** [reply]
This is what I am looking for as well

**Reply by masehdh** [reply]
would be great for flutter as well
